mmdet3d/ops/roiaware_pool3d/points_in_boxes.py

```diff
@@ -83,8 +89,12 @@ def points_in_boxes_batch(points, boxes):
     Returns:
         box_idxs_of_pts (torch.Tensor): (B, M, T), default background = 0
     """
-    assert boxes.shape[0] == points.shape[0]
-    assert boxes.shape[2] == 7
+    assert boxes.shape[0] == points.shape[0], \
+        'Points and boxes should have the same batch size'
```

Comment:
`f'Points and boxes should have the same batch size, got {boxes.shape[0]} and {points.shape[0]}'`

Reply:
got it, I will modify it